src/main/java/de/tum/in/www1/artemis/web/rest/StudentExamResource.java

```diff
@@ -240,13 +234,83 @@ public StudentExamResource(ExamAccessService examAccessService, StudentExamServi
     }
 
     /**
-     * Find the participation in participations that belongs to the given exercise that includes the exercise data
+     * GET /courses/{courseId}/exams/{examId}/studentExams/summary : Find a student exam for the summary.
+     * This will be used to display the summary of the exam. The student exam will be returned with the exercises
+     * and with the student participation and with the submissions.
+     *
+     * @param courseId  the course to which the student exam belongs to
+     * @param examId    the exam to which the student exam belongs to
+     * @return the ResponseEntity with status 200 (OK) and with the found student exam as body
+     */
+    @GetMapping("/courses/{courseId}/exams/{examId}/studentExams/summary")
+    @PreAuthorize("hasAnyRole('USER', 'TA', 'INSTRUCTOR', 'ADMIN')")
+    public ResponseEntity<StudentExam> getStudentExamForSummary(@PathVariable Long courseId, @PathVariable Long examId) {
+        long start = System.currentTimeMillis();
+        User currentUser = userService.getUserWithGroupsAndAuthorities();
+        log.debug("REST request to get the student exam of user {} for exam {}", currentUser.getLogin(), examId);
+
+        Optional<ResponseEntity<StudentExam>> courseAndExamAccessFailure = studentExamAccessService.checkCourseAndExamAccess(courseId, examId, currentUser);
+        if (courseAndExamAccessFailure.isPresent()) {
+            return courseAndExamAccessFailure.get();
+        }
+
+        // 1st: load the studentExam with all associated exercises
+        Optional<StudentExam> optionalStudentExam = studentExamRepository.findWithExercisesByUserIdAndExamId(currentUser.getId(), examId);
+        if (optionalStudentExam.isEmpty()) {
+            return notFound();
+        }
+        var studentExam = optionalStudentExam.get();
+
+        // check that the studentExam is over, otherwise /studentExams/conduction should be used
+        if (!studentExam.isSubmitted() && !studentExam.isEnded()) {
```

Comment:
Is it intended that students who haven't submitted their solution, can use this call after their exam ended? This wasn't possible before.